src/ingestion_qc/generate_html/config.vsh.yaml

```
@@ -45,8 +45,10 @@ engines:
       - type: docker
         run: |
           npm install -g pnpm@latest-10 \
-          && cd /opt && git clone https://github.com/openpipelines-bio/incubator_ingestion_qc.git \
+          && cd /opt && git clone -b ambient-rna https://github.com/openpipelines-bio/incubator_ingestion_qc.git \
```

```suggestion
          && cd /opt && git clone https://github.com/openpipelines-bio/incubator_ingestion_qc.git \
```
Let's merge in the other PR first